repose-aggregator/artifacts/valve/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     compile "org.eclipse.jetty:jetty-servlet"
     compile "org.eclipse.jetty:jetty-server"
     compile "org.eclipse.jetty:jetty-util"
+    compile "javax.inject:javax.inject"
```

Comment:
So as with our filters, we get this dependency transitively by depending on `repose-core`. Do we want to make this a `compileOnly` dependency?

I still like calling out the dependency here, but I think it was stuck at the `compile` level when we were under the impression that the container (either Valve or the container running the WAR) would provide this dependency. Since servlet containers will not necessarily provide this dependency (as it is part of JEE, but not the servlet specification), we always pull it in to core. To keep things simple, since we know core will always be available in an environment running Repose, we add it as a `compileOnly` (i.e., `provided`) dependency everywhere else it is needed.

Reply:
seems legit to me